docs/index.html

```diff
@@ -285,84 +285,140 @@ <h2>Terminology</h2>
-          (in whole or in part). A PATH might branch, combine with others, or
-          terminate prematurely. It is an abstraction that in a traceability
-          context is inherently directed, and has a declared origin node,
-          and desired end node, with each node being a change in state or location.
-          Properties may be attached to nodes via credentials and correlated to one
-          another (and as a result, to the set of items) through use of a distinct
-          PATH CORRELATION ID.
-
-          In a real world context, a PATH might be created to describe one party
-          requesting a set of goods from some other location.  The PATH can then
-          be used to group and associate the nodes as those goods move from the origination
-          point (a node) to the end destination.  Any change in state, location, or 
-          control (legal or physical responsibility of some kind for the goods) would be
-          described as a node represented by a VC which has the PATH CORRELATION ID
-          attached to it. 
-        </dd>
         <dt>
           <dfn data-lt="CORRELATION ID|CORRELATION IDs|CORRELATIONID|CORRELATIONIDs">CORRELATION ID</dfn>
         </dt>
         <dd>
-          A unique identifier assigned to a PATH complying with [[RFC4122]].
-          A CORRELATION ID MUST conform with UUID Version 4.  
-          Such an identifier is usually created upon the creation of the first
-          node to be associated with a PATH, or upon the creation of a PATH
-          (in whole or in part) after the fact.  This CORRELATION ID is attached to a
-          VC, VP, or other credentials by use of a property named "correlationId" 
-          in the credential or as the domain with a presentation.
+
+          <p>
+            A unique identifier assigned to a JSON Object complying with [[RFC4122]].
+            A CORRELATION ID MUST conform with UUID Version 4.  
+          </p>
+          <p>
+            A CORRELATION ID MAY be present in a Verifiable Credential or a Verifiable Presentation.
+          </p>
+  
           <aside class="example" title="CORRELATIONID">
             <p>
-              Use of a CORRELATION ID in a VC
+              Use of a CORRELATION ID in a JSON.
             <p>
             <pre class="js">
               {
-                "@context": [
-                  "https://www.w3.org/2018/credentials/v1",
-                  "https://www.w3.org/2018/credentials/examples/v1"
-                ],
-                "type": ["VerifiableCredential"],
-                "issuer": "did:example:Wz4eUg7SetGfaUVCn8U9d62oDYrUJLuUtcy619",
-                "correlationId": "d2dfc100-9384-49e4-b547-5e7d84ec5bf5",
-                "credentialSubject": {
-                  "originator": "Some User or System",
-                  "product": "Blueberries",
-                }...
+                "correlationId": "d2dfc100-9384-49e4-b547-5e7d84ec5bf5"
```

Comment:
goal here to make it clear this is an identifier used to group things... and describe the cases for grouping in the following sections.

- groups of definitions
- groups of instances
- groups of nodes in a path